strings/sillytips.txt

```diff
@@ -29,4 +29,4 @@ Absolutely nothing unusual will happen this shift. Promise.
 Send help. Trapped in tip writing factory.
 There is no such thing as a cow level.
 HONK.
-Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
\ No newline at end of file
+Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
```

Comment:
Whats going on here, some kind of weird end of line change?

Reply:
probably a change to CRLF

Reply:
no this is a newline at the end of the file, shouldn't be there, i'll fix it